src/embed.ts

```diff
@@ -437,5 +440,15 @@ async function _embed(
     }
   }
 
+  // Wrap view.finalize to also remove event listeners from Vega-Embed.
+  if (documentClickHandler) {
+    const originalFinalize = view.finalize;
```

Comment:
this doesn't look right - `view.finalize` is not bound to the `view` object, so calling it would have `this` be `undefined`.
```suggestion
    const originalFinalize = view.finalize.bind(view);
```